.github/workflows/maven.yml

```diff
@@ -10,8 +10,12 @@ on:
      - 'main'
 
 jobs:
+  check-versions:
+    uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/workflows/checkVersions.yml@master
+    with:
+      botName: Eclipse M2Eclipse Bot
+      botMail: m2e-bot@eclipse.org
```

Comment:
This is now in-cooperated at other places.
```suggestion
```

Reply:
reverted...